util/config.go

```diff
@@ -9,8 +9,8 @@ import (
 
 	"github.com/FactomProject/factomd/common/primitives"
 
-	llog "github.com/FactomProject/factomd/log"
-	gcfg "gopkg.in/gcfg.v1"
+	"github.com/FactomProject/factomd/log" //deleted 'llog' from beginning
```

Comment:
did you intentionally commit those changes to this file?

Reply:
These files should have been in the release, but I guess they were not.  I'm guessing the master branch was edited directly.  I'd _like_ them to be included (a young kid shadowed the development team last summer, and worked on cleaning up some lint warnings).

But they are not absolutely necessary.